AirCasting/SessionViews/SessionCartViewModel.swift

```diff
@@ -0,0 +1,47 @@
+// Created by Lunar on 29/07/2021.
+//
+
+import Foundation
+
+class SessionCartViewModel: ObservableObject {
+    private let sessionCartFollowing: SessionCartFollowing
+    @Published var isFollowing: Bool = false
+    
+    init(sessionCartFollowing: SessionCartFollowing) {
+        self.sessionCartFollowing = sessionCartFollowing
+    }
+    
+    func toggleFollowing(for session: SessionEntity) {
+        session.followedAt != nil ? sessionCartFollowing.makeFollowing(for: session) : sessionCartFollowing.makeNotFollowing(for: session)
+        switch showFollowingButton(for: session) {
+            case true: isFollowing = true
+            case false: isFollowing = false
+        }
+    }
+    
+    func showFollowingButton(for session: SessionEntity) -> Bool {
```

Comment:
I'm not sure what this function is doing. For sure not showing any buttons ;) please rename (and also make sure all the things that can be `private` in this class are)